src/client/app/translations/data.js

```diff
@@ -132,30 +131,52 @@ const localeData = {
 		"export.graph.data": "Export graph data",
 		"export.raw.graph.data": "Export raw graph data",
 		"failed.logging.in": "Failed logging in",
-		"failed.to.create.group": "Failed to create group",
 		"failed.to.create.map": "Failed to create map",
 		"failed.to.delete.group": "Failed to delete group",
 		"failed.to.delete.map": "Failed to delete map",
-		"failed.to.edit.group": "Failed to edit group",
 		"failed.to.edit.map": "Failed to edit map",
-		"failed.to.edit.meter": "Failed to edit meter",
 		"failed.to.link.graph": "Failed to link graph",
 		"failed.to.submit.changes": "Failed to submit changes",
 		"false": "False",
 		"graph.type": "Graph Type",
 		"group": "Group",
 		"group.all.meters": "All Meters",
-		"group.gps": "Group GPS (latitude, longitude)",
+		"group.area": "Group Area:",
+		"group.children.error": "You cannot have a group that has no child meters nor groups.",
```

Comment:
Maybe rephrase to "Groups must have at least one child meter or group."?

Reply:
I agree so changed.